CMakeLists.txt

```diff
@@ -42,28 +42,25 @@ option(WITH_MAEMO "Build with right click mapped to F4 (menu button)" OFF)
 ## Build
 
 set(BUILD_SHARED_LIBS OFF CACHE BOOL "")
-set(CFLAGS "${CFLAGS} -DJA2 -DGAME_VERSION=\\\"${GAME_VERSION}\\\" -DMICROINI_STATIC")
+add_definitions(-DJA2)
+add_definitions(-DGAME_VERSION="\\"${GAME_VERSION}\\"")
```

Comment:
Yes, that should not be necessary anymore since it is built in a different project...